scanner/src/lib/settings.ts

```diff
@@ -0,0 +1,6 @@
+import { env } from "$env/dynamic/private"
+
+export const settings = {
+  DEVELOPMENT: true,
```

Comment:
Now this isn't configurable

```suggestion
  DEVELOPMENT: env.DEVELOPMENT === "true",
```